### DISCUSSION TITLE
Filter for books

### DISCUSSION THREAD

**Discussion by davidszerman** [TandoorRecipes/recipes#1856] [Q&A]
Hello 

i am discovering Tandoor and I really like it so far (actually, only 1 recipe entered...)

For the books : I have a tag "Monsieur Cuisine" ; how to get all the reciptes with this tag ?

I do not find any doc on how it works

Thanks

**Comment by vabene1111** [comment]
you go to the search page and set the tag "Monsieur Cuisine", after that you hit save filter and name this filter. Now you can add the filter to any book and automatically load all recipes matching this filter.

**Comment by davidszerman** [comment]
Clear and easy. Thanks
